package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "sfx-view",
+  "version": "0.0.0",
+  "description": "SF-X View SDK containing pre-built web components for for creating e-commerce application interfaces.",
+  "private": true,
+  "workspaces": [
+    "packages/web-components/@sfx/*"
+  ],
+  "scripts": {
+    "bundle": "webpack",
+    "start": "webpack-dev-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/groupby/sfx-view.git"
```

We should standardize this between the two packages
```suggestion
    "url": "https://github.com/groupby/sfx-view.git"
```

---

Good call, I will update this.

---

Updated.